src/Sylius/Bundle/WebBundle/Behat/WebContext.php

```diff
@@ -703,6 +704,27 @@ public function iChangeMyPasswordTo($newPassword)
         $this->pressButton('Save changes');
     }
 
+    /**
+     * @When I remove :name from cart
+     */
+    public function iRemoveFromCart($name)
+    {
+        $name = explode(' ', $name)[0];
+        $trLocator = sprintf('table tbody tr:contains("%s")', $name);
+        $tr = $this->getElementByCss($trLocator);
+        $buttonLocator = sprintf('td a.btn-danger');
+        $button = $tr->find('css', $buttonLocator);
+        $button->click();
+    }
+
+    /**
+     * @Then I should not see :total near cart widget
+     */
+    public function iShouldNotSeeNearCartWidget($total)
+    {
+        $this->assertSession()->elementNotContains('css', '.masthead > ul > li.first > a > span', $total);
```

Comment:
Maybe change the view cart link title to simply 'View cart' (https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/WebBundle/Menu/FrontendMenuBuilder.php#L126) and use it as the locator instead of strange and unclear `'.masthead > ul > li.first > a > span'`?